pkg/yaml2pcl/yaml2pcl.go

```diff
@@ -342,17 +327,10 @@ func walkToPCL(v Visitor, node ast.Node, totalPCL io.Writer, suffix string) erro
 	case *ast.InfinityNode:
 	case *ast.NanNode:
 	case *ast.TagNode:
-		_, err = fmt.Fprintf(totalPCL, "%s\n", node)
+		err = walkToPCL(v, n.Value, totalPCL, "")
```

Comment:
To be honest I don't know what the proper handling of `TagNode` would be, but it has a `Value` property of type `Node` and so I thought we should recurse rather than call `String()`.  I suppose you'll need to come up with an example.